src/priyanka/WhileLoopExample.java

```diff
@@ -0,0 +1,63 @@
+package priyanka;
+
+public class WhileLoopExample {
+
+	void evenNumber(int start, int end) {
+
+		System.out.println("**************** Even numbers are ***********");
+		int index = start;
```

Comment:
why do we need third variable, you can handle this with start variable as well, think on this.
same for all methods